### PR TITLE
Restrict slack user to have only one fyle account link

### PR DESCRIPTION
**Pull request by jatzz10**
Changes -
- if user tries to link an already linked account in a different workspace, user will get a message like this -
Screenshot 2021-12-28 at 2 17 30 PM

[Clickup](https://app.clickup.com/t/1wn3qx4)

### CODE REVIEW

File: fyle_slack_app/fyle/authorization/views.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from logging import log
```
Comment (shreyanshss7) [inline fyle_slack_app/fyle/authorization/views.py:1]: Can remove this, not needed. We already have a logger in `libs`

Comment (jatzz10) [reply fyle_slack_app/fyle/authorization/views.py]: this was done

File: fyle_slack_app/fyle/authorization/views.py [hunk 2]
```diff
@@ -62,33 +62,42 @@ def get(self, request: HttpRequest) -> HttpResponseRedirect:
...
 
-                    fyle_refresh_token = fyle_utils.get_fyle_refresh_token(code)
+                fyle_user = utils.get_or_none(User, fyle_user_id=fyle_profile['user_id'])
 
-                    fyle_profile = fyle_utils.get_fyle_profile(fyle_refresh_token)
+                if fyle_user is not None:
+                    # If the fyle user already exists, send a message to user indicating they've already 
+                    # linked their Fyle account in one of their slack workspace
+                    team_name = fyle_user.slack_team.name
+                    message = f'Hey buddy you\'ve already linked your *Fyle* account in this workspace `{team_name}` :rainbow:'
```
Comment (shreyanshss7) [inline fyle_slack_app/fyle/authorization/views.py:80]: Minor message suggestion - `Hey buddy you\'ve already linked your *Fyle* account in {workspace name} workspace` 
Also, can we indicate to the user that if they want to link their account to new workspace, they should unlink from the already linked one and link in the new workspace?

**Review by shreyanshss7 — APPROVED** [review]

File: fyle_slack_app/fyle/authorization/views.py (hunk 2 above)
Comment (jatzz10) [reply fyle_slack_app/fyle/authorization/views.py]: this was done

**Pull request merged**